aten/src/ATen/native/Normalization.cpp

```diff
@@ -800,6 +800,11 @@ std::tuple<Tensor, Tensor, Tensor> _batch_norm_legit_no_stats_cpu(
     bool train, double momentum, double eps) {
   return batch_norm_cpu(self, weight_opt, bias_opt, Tensor(), Tensor(), train, momentum, eps);
 }
+std::tuple<Tensor, Tensor, Tensor> _batch_norm_legit_no_training(
+    const Tensor& self, const c10::optional<Tensor>& weight_opt, const c10::optional<Tensor>& bias_opt,
+    const Tensor& running_mean, const Tensor& running_var, double momentum, double eps) {
+  return batch_norm_cpu(self, weight_opt, bias_opt, const_cast<Tensor&>(running_mean), const_cast<Tensor&>(running_var), /*train=*/false, momentum, eps);
```

Comment:
why is this cpu only?

Reply:
Whoops 😅  will fix when i get back to my laptop.

although this code path will never actually be hit in the PT2 workflow (since the decomp for this new op is automatically opted into by inductor + any backends that run “core aten decomps”)